data/kml_test_data/generated.kml

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kml xmlns="http://earth.google.com/kml/2.2">
```

Comment:
Should the declared shema be updated to include gx extension here and in other test examples?

Reply:
This file doesn't use the gx:extension (because there are no tracks with timestamps) so the schema may be old.
Files with the gx:tracks declare this extension (generated_mixed_tracks.kml, track_with_timestamps_mismatch.kml).

Reply:
An additional check was added:
``` cpp
  std::string dataFromFileBuffer;
  {
    MemWriter<decltype(dataFromFileBuffer)> sink(dataFromFileBuffer);
    kml::SerializerKml ser(dataFromFile);
    ser.Serialize(sink);
  }
  std::string dataFromGeneratedFileBuffer;
  {
    MemWriter<decltype(dataFromGeneratedFileBuffer)> sink(dataFromGeneratedFileBuffer);
    kml::SerializerKml ser(dataFromGeneratedFile);
    ser.Serialize(sink);
  }
  TEST_EQUAL(dataFromFileBuffer, dataFromGeneratedFileBuffer, ());
}
```